src/mon/Monitor.cc

```diff
@@ -2579,43 +2578,6 @@ health_status_t Monitor::get_health_status(
     *plain += "\n";
   }
 
-  if (f && (compat || compat_warn)) {
-    health_status_t cr = compat_warn ? min(HEALTH_WARN, r) : r;
-    if (compat) {
-      f->open_array_section("summary");
-      if (compat_warn) {
-	f->open_object_section("item");
-	f->dump_stream("severity") << HEALTH_WARN;
-	f->dump_string("summary", "'ceph health' JSON format has changed in luminous; update your health monitoring scripts");
-	f->close_section();
-      }
-      for (auto& svc : paxos_service) {
-	svc->get_health_checks().dump_summary_compat(f);
-      }
-      f->close_section();
-    }
-    f->dump_stream("overall_status") << cr;
-  }
-
-  if (want_detail) {
-    if (f && (compat || compat_warn)) {
-      f->open_array_section("detail");
-      if (compat_warn) {
-	f->dump_string("item", "'ceph health' JSON format has changed in luminous. If you see this your monitoring system is scraping the wrong fields. Disable this with 'mon health preluminous compat warning = false'");
-      }
-    }
-
-    for (auto& svc : paxos_service) {
-      svc->get_health_checks().dump_detail(f, plain, compat);
-    }
```

Comment:
This block shouldn't get removed

Reply:
ah, I missed the plain text handling in that fn.  Reinstated now.